app/routes/wallet/sd_jws.py

```diff
@@ -23,16 +23,45 @@
     "/sign",
     response_model=SDJWSCreateResponse,
     summary="Sign SD-JWS",
-    description="""
-Sign Select Disclosure for JWS (SD-JWS)
-
-See https://www.ietf.org/archive/id/draft-ietf-oauth-selective-disclosure-jwt-07.html for the SD-JWT / SD-JWS spec.
-""",
 )
 async def sign_sd_jws(
     body: SDJWSCreateRequest,
     auth: AcaPyAuth = Depends(acapy_auth_from_header),
 ) -> SDJWSCreateResponse:
+    """
+    Sign Select Disclosure for JWS (SD-JWS)
+    ---
+
+    The difference between the did and verification_method fields is
+    that if the `did` field is used, acapy will make an educated guess
```

Comment:
Similar to comment about JWS route -- we refer to the attributes without introducing the endpoint